src/bento/globalStatsData.js

```diff
@@ -69,26 +69,195 @@ export const globalStatsData = [
     statIconAlt: 'A teddy bear representing participants'
   },
   {
-    statTitle: 'Samples',
-    type: 'field',
-    statAPI: 'numberOfSamples',
-    statIconSrc: samplesLogo,
-    statIconAlt: 'A test tube holder with three test tubes in the holder representing samples'
-  },
-  {
-    statTitle: 'Files',
+    statTitle: 'Studies',
     type: 'field',
-    statAPI: 'numberOfFiles',
-    statIconSrc: fileLogo,
-    statIconAlt: 'An open file drawer representing files'
+    statAPI: 'numberOfStudies',
+    statIconSrc: studiesLogo,
+    statIconAlt: 'A teddy bear representing Studies'
   },
 ];
 
 // --------------- GraphQL query - Retrieve stats details --------------
-export const GET_GLOBAL_STATS_DATA_QUERY = gql`{
-  numberOfFiles
+export const GET_GLOBAL_STATS_DATA_QUERY = gql`

```

Comment:
Looks like this query is only made for Explore page. Can you look into this one. I assume there will be different query.

Reply:
Since we don't have the global stats bar in static page we only have the query for the explore page for now , maybe in later stage if we get other pages that support the global stats bar we  might have other queries but even then we can use the same query with out giving  any variable